### ISSUE TITLE
Remove Video Link and Reference to Andrew Dupont Talk

### ISSUE THREAD

**Issue by kkruups** [rwaldron/idiomatic.js#193]
Blip Video link broken (no longer available)  see reason for broken link below at Blip TV: 

http://support.blip.tv/entries/23007795

Please remove the following video content from your github Manifesto Section or locate where video is available:

“Everything is Permitted: Extending Built-ins” by Andrew Dupont (JSConf2011, Portland, Oregon)

http://blip.tv/jsconf/jsconf2011-andrew-dupont-everything-is-permitted-extending-built-ins-5211542

**Comment by kkruups** [comment]
Rather than remove change link to youtube link, since the video is available now on youtube.

See link below.

https://www.youtube.com/watch?v=xL3xCO7CLNM

**Comment by iposton** [comment]
This link is broken. New link to Everything is Permitted: Extending Built-ins” by Andrew Dupont (JSConf2011, Portland, Oregon) is [https://www.youtube.com/watch?v=xL3xCO7CLNM](https://www.youtube.com/watch?v=xL3xCO7CLNM).

**Comment by leobalter** [comment]
@iposton would you be interested in sending a PR? Thanks!

**Comment by iposton** [comment]
Yes, I will do that. I'll send a PR later today when I have a minute. No problem.

**Comment by iposton** [comment]
Pull request submitted with new working link to  "Everything is Permitted: Extending Built-ins” by Andrew Dupont (JSConf2011, Portland, Oregon) hosted on youTube. #222
